cron-mm-upgrade-script.sh

```diff
@@ -20,22 +20,23 @@ echo -e "Latest Mattermost version: $latestVersion"
 date=$(date +'%F-%H-%M')
 
 # Establish database variables from Mattermost config.json
-DATABASE=$(cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DriverName')
-DB_USER=$(cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DataSource' | cut -d: -f2 | tr -d '/')
+DB_TYPE=$(cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DriverName')
+DB_USER=$(cat /opt/mattermost/config/config.json | jq '.SqlSettings.DataSource' | grep -o '\"[^ \:]*' | tail -c +2 | head -c -3)
```

Comment:
This doesn't work for me. Not sure if our configs are different but when I run the one I use, I get:
$: cat /opt/mattermost/config/config.json | jq --raw-output '.SqlSettings.DataSource' | cut -d: -f2 | tr -d '/'
mmuser

Running your proposed version, gives me:
$ cat /opt/mattermost/config/config.json | jq '.SqlSettings.DataSource' | grep -o '\"[^ \:]*' | tail -c +2 | head -c -3
postgres

So, original gives me the correct DB_USER of mmuser. Your proposed gives me the DB_TYPE of postgres.

Reply:
Also, when working with jq, use -r or --raw-output to remove double quotes.